AutomatedTesting/Gem/Code/Source/AutomatedTestingSystemComponent.h

```diff
@@ -39,5 +54,17 @@ namespace AutomatedTesting
         void Activate() override;
         void Deactivate() override;
         ////////////////////////////////////////////////////////////////////////
+
+        ////////////////////////////////////////////////////////////////////////
+        // Multiplayer::IMultiplayerSpawner interface implementation
+        Multiplayer::NetworkEntityHandle OnPlayerJoin(uint64_t userId, const Multiplayer::MultiplayerAgentDatum& agentDatum) override;
```

Comment:
NP: Format for these two functions is a but different. It'd be good to standardize.

Reply:
Fixed!